includes/admin_meeting.php

```diff
@@ -261,6 +261,26 @@ function () {
             </div>
         <?php } ?>
 
+		<div class="meta_form_row">
+			<label for="timezone">
+				 <?php _e('Timezone', '12-step-meeting-list') ?>
+			</label>
+
+            <select name="timezone" id="timezone">
+				<?php
+				$selected_timezone = isset($meeting->timezone) ? $meeting->timezone : ''; // Allow null tz for backwards compatibility
+				$timezones = DateTimeZone::listIdentifiers(); 
+				foreach ($timezones as $timezone) {
+        echo '<option value="' . htmlspecialchars($timezone) . '"';
+        if ($timezone === $selected_timezone) {
+            echo ' selected';
```

Comment:
to be consistent with other dropdowns, it would be great to use wordpress's `selected` function: https://developer.wordpress.org/reference/functions/selected/